codegen/service/service.go

```diff
@@ -143,12 +143,12 @@ type Service interface {
 {{- range .Methods }}
 	{{ comment .Description }}
 	{{- if .ViewedResult }}
-		{{ printf "%s must return one of the following views" .VarName | comment }}
+		{{ comment "The return value must have one of the following views" }}
```

Comment:
There is more than one return value and it may be good to be explicit so maybe something like:
```
The "view" return value must have one of the following values:
```